data/campaigns/Sceptre_of_Fire/scenarios/1_A_Bargain_is_Struck.cfg

```diff
@@ -187,7 +187,7 @@
         # the background images are placeholders from the core assets
         [part]
             # wmllint: local spelling Ay
-            story= _ "Ay, the Sceptre of Fire. The sceptre has a long, glorious, and fearful history. But I am not here to tell you how the jewel within it was brought over the ocean by Haldric the Great. Nor will I tell the story of Garard I, and the challenge he set, or that of Konrad, King of Wesnoth by virtue of the Sceptre."
+            story= _ "Ay, the Sceptre of Fire. The sceptre has a long, glorious, and fearful history. But I am not here to tell you how the jewel within it was brought over the ocean by Haldric the Great; nor will I tell the story of Garard I, and the challenge he set; nor that of Konrad, King of Wesnoth by virtue of the Sceptre."
```

Comment:
These should not be semicolons, they should be commas. The second clause (the first "nor" clause) in that big sentence could have the comma removed before "and".

Reply:
@max-torch @nemaara Unrelated: I noticed this just as I'm starting SoF afresh in 1.16.1 release.

> The sceptre has a long, glorious, and fearful history.

Can we capitalise Sceptre in this sentence, please? Reasons:
1. It's clearly referring to the Sceptre of Fire specifically.
2. It's consistent with capitalised Sceptre in the first and last sentences of this paragraph.

Of if you prefer, I'll just make the change in a separate commit.

Reply:
I can do it.

Reply:
Thank you. Marking as resolved since I see you've satisfied nemaara's request about commas vs semi-colons as well.